src/plugins/action_increment_stage/src/incrementStage.ts

```diff
@@ -0,0 +1,93 @@
+import { ActionPluginOutput } from '../../types'
+
+type IParameters = {
+  applicationId: number
+}
+
+type Stage = {
+  id: number
+  number: number
+  title: string
+}
```

Comment:
Consider moving this to the `types` file

Reply:
Hmm, I thought this one would probably be unique to this plugin. How about moving it once we use it in another plugin?